packages/11ty/.gitignore

```diff
@@ -19,6 +19,9 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+# macOS
+.DS_Store
+
```

Comment:
@hbalenda may have addressed adding `.DS_Store` to the ignore patterns in a separate pull-request.